docs/UserGuide.md

```diff
@@ -10,17 +10,22 @@ to organize the records of their patients.
 
 ## Table of Contents
 
-* [Quickstart](#quickstart)
-* [Features](#features)
-    * [Print a help message: `help`](#print-a-help-message-help)
-    * [Adding a patient: `add`](#adding-a-patient-add)
-    * [Listing all patients: `list`](#listing-all-patients-list)
-    * [Loading a patient's medical records: `load`](#loading-a-patients-medical-records-load)
-    * [Displaying the current loaded patient: `current`](#displaying-the-current-loaded-patient-current)
-    * [Recording a patient's consultation details: `record`](#recording-a-patients-consultation-details-record)
-    * [Retrieving a patient's consultation details: `retrieve`](#retrieving-a-patients-consultation-details-retrieve)
-    * [Exiting the program: `exit`](#exiting-the-program-exit)
-* [Command summary](#command-summary)
+- [Patient Manager User Guide](#patient-manager-user-guide)
```

Comment:
Not much of an issue, but the [markdown coding standard](https://se-education.org/guides/conventions/markdown.html) seems to prefer `*` over `-` for bullet points

Reply:
Thanks! It has been updated.